### DISCUSSION TITLE
Cannot find module './amp-mode' / failed to collect page data

### DISCUSSION THREAD

**Discussion by zorarkdouble8** [vercel/vercel#8723] [Help]
Here are the logs when I try to deploy my application:
![image](https://user-images.githubusercontent.com/65358496/195464653-67ee5ce5-c01a-42ca-9c51-2822ee8a10f5.png)

I have no idea what the module amp-mode is and along with that, it fails to collect page data for "/usr/local/share/.config/yarn/global/node_modules/next/dist/build/utils.js:916:15" and I use npm and not yarn. So in all, I'm very confused and I have no idea how to fix this error.

Another thing is I ran "npx next build" in vs code and it builds without any errors.

Also, if you need more info about the application, I'm happy to provide.

**Comment by amyegan** [comment]
Sorry you're getting an error @zorarkdouble8! Please share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so we can get a closer look at what is going wrong.

**Reply by zorarkdouble8** [reply]
So, I've found the problem and it's git not including all of the files for the next node module

**Reply by amyegan** [reply]
Oh no! I'm glad you figured out what was causing it and I hope you have found an easy solution. Thanks for coming back here to share what you found!
